admin/cluster_cleanup.py

```diff
@@ -0,0 +1,76 @@
+
+import sys
+
+from itertools import repeat
+
+from twisted.internet.defer import inlineCallbacks
+from twisted.python.filepath import FilePath
+from twisted.python.usage import Options, UsageError
+
+from flocker.apiclient import FlockerClient
+from flocker.common import gather_deferreds, loop_until
+from flocker.control.httpapi import REST_API_PORT
+
+
+class ScriptOptions(Options):
+    optParameters = [
+        ['control-node', None, None,
+         "The address of the cluster's control node"],
+        ['cert-directory', None, None,
+         "The directory containing the cluster certificates"],
+    ]
+
+    def postOptions(self):
+        if not self['control-node']:
+            raise UsageError("Control node address must be provided.")
+        if not self['cert-directory']:
+            raise UsageError("Certificates directory must be provided.")
+
+
+@inlineCallbacks
+def main(reactor, args, base_path, top_level):
+    try:
+        options = ScriptOptions()
+        options.parseOptions(args)
+    except UsageError as e:
+        sys.stderr.write(e.args[0])
+        sys.stderr.write('\n\n')
+        sys.stderr.write(options.getSynopsis())
+        sys.stderr.write('\n')
+        sys.stderr.write(options.getUsage())
+        raise SystemExit(1)
+
+    certificates_path = FilePath(options['cert-directory'])
+    cluster_cert = certificates_path.child(b"cluster.crt")
+    user_cert = certificates_path.child(b"user.crt")
+    user_key = certificates_path.child(b"user.key")
+    client = FlockerClient(reactor, options['control-node'], REST_API_PORT,
+                           cluster_cert, user_cert, user_key)
+
```

Comment:
Put the code below this point into a separate function, as it is independently useful outside of the script.